vere/reck.c

```diff
@@ -242,7 +192,31 @@ _reck_kick_newt(u3_noun pox, u3_noun fav)
       u3_ames_ef_turf(u3k(u3t(fav)));
       u3z(pox); u3z(fav); return c3y;
     } break;
+
+    case c3__west: {
+      u3_noun who, cha, dat;
+      u3x_trel(u3t(fav), &who, &cha, &dat);
+
+      //  XX route by cha path?
+      //  s/b //give/prox
+      //
+      switch ( u3h(dat) ) {
+        default: break;
+
+        case c3__that: {
+          u3_http_ef_that(u3k(who), u3k(u3t(dat)));
+          u3z(pox); u3z(fav); return c3y;
+        }
+      }
+    }
+
+    case c3__woot: {
+      //  XX print tang if nack?
```

Comment:
Printing on nack does seem like a good idea, unless it happens all the time under normal conditions. I imagine it's not crucial. Maybe it's something to revisit once we have a better logging system.